event-app-ui/src/components/EditProfile/EditProfile.jsx

```diff
@@ -0,0 +1,87 @@
+import "components/EditProfile/EditProfile.css";
+import PropTypes from "prop-types";
+import axios from "axios";
+import { useState } from "react";
+
+EditProfile.propTypes = {
+  setOpenProfile: PropTypes.func.isRequired,
+  fullname: PropTypes.string.isRequired,
+  email: PropTypes.string.isRequired,
+  tel: PropTypes.string.isRequired,
+  username: PropTypes.string.isRequired,
+};
+
+export default function EditProfile(props) {
+  const { setOpenProfile, fullname, email, tel, username } = props;
+  const [newFullname, setNewFullname] = useState(fullname);
+  const [newEmail, setNewEmail] = useState(email);
+  const [newTel, setNewTel] = useState(tel);
+  const [error, setError] = useState(false);
```

Comment:
The error handling in this component can be improved. You have error state that only gets set to true when the user submits, but you don't ever set it back to false. If the user tried to submit with an empty field, and then they go back and fill in that field, the label will still say  there's an error, right?

Ideally, you set a variable that calculates whether newFullname, newTel, and newEmail are empty. Then disable the submit button if the variable is true. Then in your submit function you just check that variable again before submitting the request.